worlds/noita/Regions.py

```diff
@@ -15,10 +15,20 @@ def create_region(world: MultiWorld, player: int, region_name: str) -> Region:
         location = Locations.NoitaLocation(player, location_name, location_data.id, new_region)
         opt_orbs = world.orbs_as_checks[player].value
         opt_bosses = world.bosses_as_checks[player].value
+        opt_paths = world.path_option[player].value
         ltype = location_data.ltype
         flag = location_data.flag
-        if flag == 0 or ltype == "orb" and flag <= opt_orbs or ltype == "boss" and flag <= opt_bosses:
+        if flag == 0 and ltype != "orb" or "boss":

```

Comment:
```py
if flag == 0 and ltype != "orb" and ltype != "boss":
```

I'm assuming this is what you meant?

Reply:
Oh, right, order of operations, duh. Can I do something like `if flag == 0 and ltype != ("orb" or "boss")`?

Reply:
No, it's more that you're not telling it the full statement. You're writing it in a way that would be intuitive in english but it's a computer so it doesn't work like that. 

This is how the original would be evaluated:

1: `if flag == 0 and ltype != "orb" or "boss":`
2: `if True and ltype != "orb" or "boss":`
3: `if True and False or "boss":`
4: `if False or "boss":` (and evaluated first)
5: `if "boss":` 
6: `if True:` (strings are truthy)


The second one:

1: `if flag == 0 and ltype != ("orb" or "boss")`  (because `"orb"` is truthy it chooses that)
2: `if flag == 0 and ltype != "orb"`
3: `if True and ltype != "orb"`
4: `if True and False`